lib/predicator/evaluator.rb

```diff
@@ -36,6 +36,22 @@ def process instruction
         stack.push context[instruction.last]
       when "to_bool"
         stack.push !!stack.pop
+      when "to_int"
```

Comment:
I think we could extract the body of these case statements (this one and the `to_str`) into methods that are called (e.g. `stack.push to_int(stack.pop)`